Makefile.builders

```diff
@@ -0,0 +1,5 @@
+BUILDER_ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+
+.PHONY: wasi-builder-16
+wasi-builder-16:
+	docker build --build-arg WASI_SDK_VERSION=16 -f ${BUILDER_ROOT_DIR}/Dockerfile.wasi-builder -t ghcr.io/vmware-labs/wasi-builder:16 ${BUILDER_ROOT_DIR}
```

Comment:
Shouldn't this be tagged in projects.registry.vmware.com/wasmlabs/containers instead of ghcr.io ?

Reply:
I realized with this work that we can publish on ghcr.io already (https://github.com/orgs/vmware-labs/packages/container/package/wasi-builder) and (https://github.com/orgs/vmware-labs/packages/container/package/php-builder).

I think we might want to discuss it. If we are going to work with GitHub workflows and automate stuff on GitHub I think it might be more convenient to deploy assets on the GitHub registry, given it makes authentication & authorization easier and more integrated.